sdk/monitor/monitor-query/src/internal/util.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { QueryLogsOptions } from "../models/logsModels";
+
+/**
+ * @internal
+ */
+export function formatPreferHeader(
+  args: Pick<QueryLogsOptions, "serverTimeoutInSeconds" | "includeQueryStatistics"> | undefined
+): { Prefer: string } | undefined {
+  if (args == null) {
```

Comment:
feels a little weird since the input is `| undefined` and not `| null` and since it's an object type you could just use a falsy check, right?

Reply:
Added a falsy check.